test/case/ietf_system/add_delete_user.py

```diff
@@ -2,11 +2,12 @@
 
 import infamy
 import copy
-import crypt
+from passlib.hash import sha256_crypt
```

Comment:
You need to bump the container version at the top of `test/env` in this PR as well. Otherwise passlib will not be found.

(I'm a bit delayed pushing the new container from PR #89 due to network issues at the office. Sorry!)

Reply:
I forgot to rebase after the previous PR.
Pushed again

Reply:
I've just (finally) pushed v0.6 of the container as well, so hopefully things will go much smoother this time :-)